packages/plugin/package.json

```diff
@@ -27,7 +28,8 @@
     "docs": "theiaext docs"
   },
   "devDependencies": {
-    "@theia/ext-scripts": "^0.3.16"
+    "@theia/ext-scripts": "^0.3.16",
+    "typedoc": "^0.13.0"
```

Comment:
we already have dev dependencies to it in root package: https://github.com/theia-ide/theia/blob/26dea592e695bfb70404294d301b7d82f8615b3d/package.json#L41

It would be nice to manage it in one place.

Reply:
the issue is that to make it global https://github.com/TypeStrong/typedoc/issues/844 needs to be in a release.

Reply:
It would be fine to update typedoc version in the root.

Reply:
if I update in root then it will fail for all modules (except packages/plugin) due to the bug I referenced. This is why I was wondering

Reply:
ok, strange, from the issue I've derived that it is fixed in `0.13.0`

Reply:
hmm no, fix has been committed there https://github.com/TypeStrong/typedoc/commit/c03ac2d3dcc538766ae895b8cd68da1d74b65060 (26 october) and `0.13.0` has been tagged 10 October :-/